client/src/utils/getToken.ts

```diff
@@ -1,4 +1,4 @@
-const getToken = () => {
+const getToken = () : Token => {
```

Comment:
Token is already defined in Login and Signup. Not necessary to define here since outcome of function is obvious.

Reply:
good point, done